README.md

```diff
@@ -207,11 +179,29 @@ public boolean callH(String dest, Map<String, String> headers);
 To enable Pushkit Integration in the SDK:
 Login with registerToken, create FirebaseMessagingService class and implement relayVoipPushNotification method
```

Comment:
instead of `relayVoipPushNotification` we can mention `loginForIncomingWithUsername`,  and `loginForIncomingWithJwt`